### PR TITLE
Fix for broken SDK 25

### PR DESCRIPTION
**Pull request by rjaros87**
@gildegoma  @brantone, please review. I don't test it because I have newest ChefDK

Fix for GH-34

### CODE REVIEW

**Comment by gildegoma** [comment]
@rjaros87 I'll have time this afternoon/evening to fully review it, and we hopefully can produce the release before next week.

File: recipes/default.rb [hunk 1]
```diff
@@ -33,6 +33,7 @@
 # Install required libraries
 #
 if node['platform'] == 'ubuntu'
+  package 'unzip'
```
Comment (gildegoma) [inline recipes/default.rb]: Why do you add this explicit package dependency? This is [transitively covered](https://github.com/chef-cookbooks/ark/blob/7559940dc37cfd672178d15571b7178f267178cb/attributes/default.rb#L32) by the `ark` cookbook.

File: metadata.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
 maintainer_email 'foss@gilles.cornu.name'
 license 'Apache 2.0'
 description 'Installs Google Android SDK'
-version '0.2.0'
+version '0.2.1'
```
Comment (gildegoma) [inline metadata.rb]: It would be great if you also update the `CHANGELOG.md`...

Comment (gildegoma) [inline metadata.rb]: Also need to bump the version number in [`metadata.rb`](https://github.com/gildegoma/chef-android-sdk/blob/master/metadata.rb#L6)

File: recipes/default.rb (hunk 1 above)
Comment (rjaros87) [inline recipes/default.rb]: ok

File: metadata.rb (hunk 2 above)
Comment (rjaros87) [inline metadata.rb]: there is :) see above

Comment (gildegoma) [inline metadata.rb]: Oops my bad 😬 sorry! Better that I wait for this afternoon, when I'll really have time to focus on review/test work ;-)
See you later then... Thanks again!

**Comment by Brantone** [comment]
Looks good to me 👍

**Comment by gildegoma** [comment]
Closing as I'll reuse this commit "as is" and open an new pull request this evening. 

@rjaros87 please don't force push your commit.

**Comment by rjaros87** [comment]
@gildegoma ok.
I think you should focus only on the hotfix. Problems with Travis/Chef12 etc. I resolved in #33

**Comment by gildegoma** [comment]
@Brantone @rjaros87 please review 759feaa4415bc5598811b7f2e6506052e4c1c04b.

**Pull request closed without merging**